chains/boba/logo.svg

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 27.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 1080 1080" style="enable-background:new 0 0 1080 1080;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#ACD905;}
+	.st1{fill:#AEAFAA;}
+	.st2{fill:#5D6E3C;}
+	.st3{fill:#FFFFFF;}
+	.st4{fill:#343434;}
+	.st5{fill:#8A8A8A;}
+	.st6{fill:#3C3C3B;}
+	.st7{fill:#141414;}
+	.st8{fill:#393939;}
+</style>
+<g>
+	<g>
+		<g>
+			<path class="st0" d="M653.7,651.8H347.6V204.5h306.1c105.1,0,190.2,85.2,190.2,190.2v66.8C843.9,566.6,758.8,651.8,653.7,651.8"
+				/>
+			<path class="st1" d="M732.6,875.5h-385V651.8c0-123.5,100.1-223.6,223.6-223.6h161.4c123.5,0,223.6,100.1,223.6,223.6
+				C956.2,775.3,856.1,875.5,732.6,875.5"/>
+			<path class="st2" d="M843.9,457.9v3.7c0,105.1-85.1,190.2-190.3,190.2H347.6c0-61.7,25.1-117.6,65.5-158.1
+				c40.5-40.5,96.4-65.5,158.2-65.5h161.3C773.2,428.2,811.2,439,843.9,457.9"/>
+		</g>
+		<g>
+			<g>
+				<circle class="st3" cx="348.2" cy="540" r="224.5"/>
+			</g>
+			<g>
+				<polygon class="st4" points="348.2,343.7 345.6,352.7 345.6,612.2 348.2,614.9 468.7,543.6 				"/>
+				<polygon class="st5" points="348.2,343.7 227.7,543.6 348.2,614.9 348.2,488.9 				"/>
+				<polygon class="st6" points="348.2,637.7 346.7,639.5 346.7,731.9 348.2,736.3 468.8,566.5 				"/>
+				<polygon class="st5" points="348.2,736.3 348.2,637.7 227.7,566.5 				"/>
+				<polygon class="st7" points="348.2,614.9 468.7,543.6 348.2,488.9 				"/>
+				<polygon class="st8" points="227.7,543.6 348.2,614.9 348.2,488.9 				"/>
+			</g>
+		</g>
+	</g>
+</g>
+</svg>
```

Comment:
Hey @boyuan-chen thank you for your PR to the Hyperlane Registry! Can you please crop and minify the SVG file to conform to the standard used in the repo? You can use these tools to do it:
- crop: https://svgcrop.com/
- minify: https://jakearchibald.github.io/svgomg/

Reply:
Fixed!